src/adapter/dto/blog_dto.py

```
@@ -0,0 +1,12 @@
+from pydantic import BaseModel, Field
+from typing import List, Dict
```

Ruff (0.8.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)